Source/Moya.swift

```diff
@@ -150,6 +150,12 @@ public extension MoyaProvider {
     public final class func DefaultRequestMapping(endpoint: Endpoint<Target>, closure: NSURLRequest -> Void) {
         return closure(endpoint.urlRequest)
     }
+
+    public final class func DefaultAlamofireManager() -> Manager {
+        let manager = Alamofire.Manager.sharedInstance
+        manager.startRequestsImmediately = false
```

Comment:
Yeah.. I see what you mean. Me too actually, I'm not sure about another option. 

Either we find another way to build an `Alamofire.Request` ourself which is obviously not a good idea, or we can just pass a `NSURLRequest` instance into plugin's delegate method instead of `Alamofire.Request`. In latter case, we gain a safer plugin mechanism (as I mentioned [here](https://github.com/Moya/Moya/issues/326#issuecomment-162242355)), but we will lose a little flexibility, it'll break `CredentialsPlugin`.

 ¯\\\_(ツ)\_/¯ , still seeking ...